en/docs/quick-starts/oidc-session-management.md

```diff
@@ -0,0 +1,126 @@
+        sample you can only test OIDC for the Authorization Code flow)
+    -   **Client Id:** Enter the client ID (OAuth Client Key under
+        Registering the relying Application, step 4) of the registered
+        playground2 application
+    -   **Scope:** `            openid           `
+    -   **Callback URL:**
+        `            http://wso2is.local:8080/playground2/oauth2client           `
+    -   **Authorize Endpoint:**
+        `            https://localhost:9443/oauth2/authorize           `
+    -   **Logout Endpoint:**
+        `            https://localhost:9443/oidc/logout           `
+    -   **Session Iframe Endpoint:**
+        `            https://localhost:9443/oidc/checksession?client_id=<clientID of playground2 application>           `
+
+
+2.  Log in with the user credentials and click **Approve Always** at the
+    consent page.
+
+    In order to enable single logout (SLO) you must use the **Approve
+    Always** option. If this is not done, the passive request will not
+    work, and without passive requests, the SLO protocol will not work. 
+    Do this to avoid errors during execution.
+
+    <img name='playground-login' src='../../assets/img/samples/playground-login.png' class='img-zoomable'/>
+
+3.  Once it is successfully authenticated, the OpenIDConnect
+    Provider(OP) will redirect back to the client application with the
+    authorization code and the session state. You can see this in the
+    logs of the console, as seen below.
+    
+4.  Enter the following values and click **Get Access Token** to receive
+    the ID token and access token.  
+    -   **Callback URL:**
+        `                         http://wso2is.local:8080/playground2/oauth2client                       `
+    -   **Access Token Endpoint:**
+        `                         https://localhost:9443/oauth2/token                       `
+    -   **Client Secret:** Enter the client secret of playground2
+        application
+        
+    <img name='authorization-code' src='../../assets/img/samples/authorization-code.png' class='img-zoomable'/>
+    
+5.  You will receive the access token. You can also enter the **UserInfo
+    Endpoint** as
+    `                     https://localhost:9443/oauth2/userinfo                  `
+    to use the received access token to obtain user claims if needed.  
+    <img name='access-token' src='../../assets/img/samples/access-token.png' class='img-zoomable'/> 
+    
+6.  Access the following URL on a separate window of the browser, and
+    click on **Import Photos:**
+    `          http://wso2is.local:8080/playground2/         `
+    
+    !!! note 
+        For this scenario we need two relying party applications.
+        To do this, make a copy of the playground2.war file that
+        was generated when you set up the sample webapp, 
+        and rename it as playground3.war in the same location. 
+        Repeat the above steps for the playground3 application.
```

Comment:
the steps under this section including how to deploy and register an SP for playground app are in the "quick-starts/auth-code-playground.md file already...shall we link to that directly or we can use it as a fragment?

Reply:
This section contains some additional info  like Session Iframe Endpoint